tests/ssl-opt.sh

```diff
@@ -12671,8 +12671,7 @@ run_test    "TLS 1.3: Check server no suitable signature algorithm, G->m" \
                                  --x509certfile data_files/server2-sha256.crt --x509keyfile data_files/server2.key \
                                  --priority=NORMAL:-SIGN-ALL:+SIGN-RSA-SHA512:+SIGN-RSA-PSS-RSAE-SHA512:+SIGN-ECDSA-SECP521R1-SHA512"  \
             1 \
-            -s "ssl_tls13_pick_key_cert:selected signature algorithm rsa_pss_rsae_sha512" \
-            -s "select_sig_alg_for_certificate_verify:no suitable signature algorithm found"
+            -S "ssl_tls13_pick_key_cert:check signature algorithm"
```

Comment:
Sorry, I don't understand this. Why is this the right thing to assert now? (Not saying it's wrong, just that I struggle to understand it, and would appreciate a hint.) (Edit: 3 test cases with this.)

Reply:
Function that prints these messages has been removed (`ssl_tls13_select_sig_alg_for_certificate_verify`), but now maybe we could use the following instead:
```
-s "CertificateVerify signature failed"
-S "CertificateVerify signature with"
```

Reply:
In the test case, there is no suitable signature algorithm to compute the signature for the CertificateVerify message: the client and the server do not have a signature algorithm in common suitable for CertificateVerify: PKCS1 algorithms are not allowed for CertificateVerify signatures. Thus there is no signature algorithm to check against a certificate and we check that the message "ssl_tls13_pick_key_cert:check signature algorithm" is not present.

Reply:
Ok, thanks for the explanation!